src/objects/aff/zcl_abapgit_object_common_aff.clas.abap

```diff
@@ -579,9 +589,37 @@ CLASS zcl_abapgit_object_common_aff IMPLEMENTATION.
           RECEIVING
             result = lv_json_as_xstring.
 
+        " ABAP Language Version should not be serialized, remove it from lv_json_as_xstring
```

Comment:
You can use `zcl_abapgit_convert`, `string_to_xstring` `xstring_to_string` for this part

Reply:
alright, thanks